front-end/src/app/app.module.ts

```diff
@@ -4,10 +4,10 @@ import { NgModule } from '@angular/core';
 import { AppComponent } from './app.component';
 import { QuestionComponent } from './question/question.component';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-import { MatInputModule} from '@angular/material/input';
-import {FormsModule, ReactiveFormsModule} from "@angular/forms";
-import {HttpClientModule} from "@angular/common/http";
-import {MatButtonModule, MatCardModule} from '@angular/materialbutton';
+import { MatInputModule } from '@angular/material/input';
+import { FormsModule, ReactiveFormsModule } from "@angular/forms";
+import { HttpClientModule } from "@angular/common/http";
+import { MatButtonModule, MatCardModule } from '@angular/materialbutton';
```

Comment:
remove button word after material, after changes this should be '@angular/material'